### ISSUE TITLE
Show bell even on sign-out status. 

### ISSUE THREAD

**Issue by MoeNick** [Giveth/giveth-dapps-v2#1889]
As a user when I still dod not signed in my wallet I can be able to see the bell icon with total notification count, the bell should be clickable and after click it ask me to sign-in.

**Comment by MoeNick** [comment]
@maryjaf pls keep a record of this! :)

**Comment by maryjaf** [comment]
I tested this on notification branch
When I tap on sign out, the notification count is still shown but when I tap on bell icon ask me to sign-in

**Comment by alireza-sharifpour** [comment]
I think it's related to #1860 issue, am i right @MoeNick ?
With current implementation we show the counter to the user only if the wallet is connected no matter whether he is Signed in or Signed out.

**Comment by MoeNick** [comment]
@alireza-sharifpour @MohammadPCh Please redirect usr to notification page after the sign in

**Comment by alireza-sharifpour** [comment]
> @alireza-sharifpour @MohammadPCh Please redirect usr to notification page after the sign in

I tested it and we are currently redirecting the user to the notifications page after signing in.
Is it not working for you? @MoeNick

**Comment by MohammadPCh** [comment]
Yeah, I implemented it when I was working on other menus. Thanks, @alireza-sharifpour .
